controller/api/destination/server_test.go

```diff
@@ -760,6 +850,44 @@ func TestGetProfiles(t *testing.T) {
 			t.Fatalf("Expected port %d to be an opaque protocol, but it was not", opaquePort)
 		}
 	})
+
+	t.Run("Return profile with unknown protocol hint and identity when service name contains skipped inbound port", func(t *testing.T) {
```

Comment:
Getting profile for pod IP here, so maybe reword:
```suggestion
	t.Run("Return profile with unknown protocol hint and identity when pod contains skipped inbound port", func(t *testing.T) {
```